mise/tasks/test

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
-# mise description="Test the project using Swift Package Manager"
+# mise description="Test all packages"
 
 set -euo pipefail
 
-swift test --package-path $MISE_PROJECT_ROOT
+echo "Testing CLI package..."
+swift test --package-path $MISE_PROJECT_ROOT/packages/cli
```

Comment:
I wonder if we can use [file task dependencies](https://mise.jdx.dev/tasks/file-tasks.html#task-configuration) to shell out to existing scripts for the sub-packages.